Sources/SearchableCollectionViewController/SearchBarView.swift

```diff
@@ -1,34 +1,36 @@
 import Foundation
 import UIKit
 
-class SearchBarView: UICollectionReusableView {
-    
-    static let height: CGFloat = 44.0
-    static let kind = String(describing:SearchBarView.self)
-    static let defaultSearchBackgroundColor = UIColor(hex: "00C365")
-    
-    dynamic open var searchBackgroundColor: UIColor? = SearchBarView.defaultSearchBackgroundColor {
-        didSet { searchBar.backgroundColor = searchBackgroundColor }
+open class SearchBarView: UICollectionReusableView {
```

Comment:
I don't think `SearchBarView` is meant to be open since it's not meant to be modified outside of SearchableCollectionController by subclassing. Instead, we provide UIAppearance proxy methods.